crates/libs/bindgen/src/metadata.rs

```diff
@@ -475,6 +475,10 @@ fn method_def_last_error(row: MethodDef) -> bool {
     }
 }
 
+pub fn method_def_is_noexcept(method: MethodDef) -> bool {
+    method.has_attribute("NoExceptionAttribute")
```

Comment:
The special "remove" methods for event handlers are also `noexcept` in C++/WinRT. I'll add that as a separate update since that introduces a lot more (generated) code changes in the `windows` crate.